src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -471,12 +471,8 @@ private function renderDisableMultistorePage(int $productId = null): Response
     {
         return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/disabled.html.twig', [
             'errorMessage' => $this->trans(
-                'This page is only compatible in a single store context. To access the page, please select a store or [1]disable the multistore feature[/1].',
-                'Admin.Notifications.Info',
-                [
-                    '[1]' => sprintf('<a href="%s">', $this->get('router')->generate('admin_preferences')),
-                    '[/1]' => '</a>',
-                ]
+                'This page isn\'t available when working on a group of stores. Switch context to work on it.',
```

Comment:
The issue you are referring to is quite old It was related to legacy page
This is v2 page and the current wording has been validated by @Julievrz so I think we should close this PR but I let her validate this

Reply:
@Progi1984 So, as discussed with Jonathan, the right wording is "This page is only compatible in a single store context. To access the page, please select a store or [1]disable the multistore feature[/1]."

I thought the wording had changed again, sorry. 😅

Reply:
No problem. I close the PR and the issue.